signer/src/stacks/contracts.rs

```diff
@@ -706,8 +710,7 @@ impl AsContractCall for AcceptWithdrawalV1 {
     ///  8. That the fee matches the expected assessed fee for the output.
     ///  9. That the first input into the sweep transaction is the signers'
     ///     UTXO.
-    /// 10. That the signer bitmap matches the bitmap from our records.
-    /// 11. That the withdrawal request is not already completed.
+    /// 10. That the withdrawal request is not already completed.
```

Comment:
nit: there still a `11. ...` comment right below

Reply:
oops. fixed in https://github.com/stacks-network/sbtc/pull/1508/commits/d3e172867dd00d69c0803e391891820894c1073f